engine/execution/provider/engine.go

```diff
@@ -158,55 +158,44 @@ func (e *Engine) onChunkDataRequest(
 	// extracts list of verifier nodes id
 	chunkID := req.ChunkID
 
-	log := e.log.With().
+	lg := e.log.With().
```

Comment:
I quite like `log` here

Reply:
fixed in https://github.com/onflow/flow-go/pull/968/commits/7e777048874f17c042ada6867194b6e60d3555cd. The original refactoring reason was to avoid confusion with the package name `log`.

Reply:
I actually like this change. 

I ran into a few problem that I used `log.Info().Msg("some log...")`, but forgot to define the log with `log := e.log.With()...`

Then go-fmt would automatically import zerolog:
```
	"github.com/rs/zerolog/log"
```

This caused the log to lose all the context.

using `lg` prevents such mistake. Because if you forgot to define `lg := e.log.With()...`, then compiler would error

Reply:
@m4ksio so can I revert it back according to Leo's comment?

Reply:
If you think thats proper way, go ahead

Reply:
refactored back in https://github.com/onflow/flow-go/pull/968/commits/ad14e2fb59a0478a4e30e91b65fee79a518d96e0.